auto_py_to_exe/web/js/i18n.js

```diff
@@ -400,6 +432,7 @@ const translationMap = {
                 tr: 'ARGÜMANLAR',
                 th: 'อาร์กิวเมนต์',
                 fr: 'ARGUMENTS',
+                it 'OPZIONI',

```

Comment:
Looks like this is the issue: you're missing a `:`

Reply:
Oops my bad! Fixed in the latest commit.